WordPress/Classes/Models/AbstractPost.m

```diff
@@ -581,6 +581,12 @@ - (BOOL)hasRemoteChanges
             || self.remoteStatus == AbstractPostRemoteStatusFailed);
 }
 
+- (BOOL)hasVersionConflict
+{
+    AbstractPost *original = (AbstractPost *)self.original;
```

Comment:
I'm just wondering here: I think we can have a scenario when the user has a draft in which the date is more "recent" than the `original`.

That wouldn't configure a version conflict. Wdyt?

Reply:
I don't think you can, because you can't currently save the post if there's a conflict. Let me try to make one happen.

Reply:
I tested here with a published post and I'm seeing "Version Conflict" while there's no version conflict.

Steps to reproduce:

1. Create and publish a post
2. Open this post, add some content and close the app
3. Open the app and check the post list, this post will be displayed with the "Version Conflict" label

That's because we created a new draft, with a recent date. Recent date != post.original date.